src/extension.ts

```diff
@@ -255,11 +255,11 @@ export async function activate(context: vscode.ExtensionContext) {
                 try {
                     if (!("eth-wake" in pipxList.venvs)) {
                         await pipxInstall(outputChannel);
-                        pipxList = JSON.parse(execFileSync("pipx", ["list", "--json"]).toString("utf8").trim());
+                        pipxList = JSON.parse(execaSync("pipx", ["list", "--json"]).stdout.trim());
                     }
 
                     for (const appPath of pipxList.venvs["eth-wake"].metadata.main_package.app_paths) {
-                        if (appPath.__Path__.endsWith("wake")) {
+                        if (path.parse(appPath.__Path__).name === "wake") {
```

Comment:
The `wake` executable installed through pipx may be named `wake.exe` on Windows. `path.parse(...).name` should return just the name of the file without an extension.